### ISSUE TITLE
recent dependency update breaks jsdoc for node < 14.14.0 when 'engine-strict=true'

### ISSUE THREAD

**Issue by BrianFreedman** [jsdoc/jsdoc#1972]
[this change](https://github.com/jsdoc/jsdoc/commit/a0240542cfbb8ee1cdc1f0770d584872915193f1) updates the version of klaw to 4.0.1 which requires node >= 14.14.0

ref: https://github.com/jprichardson/node-klaw/blob/04796a92d6a2d8642eb9d208d1d9e03972214ca3/package.json#L23

### Input code

add a `.npmrc` file to a project with `engine-strict=true`

### JSDoc configuration

n/a

### JSDoc debug output

n/a

### Expected behavior

should be able to install jsdoc

### Current behavior

```
root@ea327157d1ce:/src/app/temp# echo "engine-strict=true" > .npmrc
root@ea327157d1ce:/src/app/temp# cat .npmrc
engine-strict=true
root@ea327157d1ce:/src/app/temp# npm install jsdoc
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for klaw@4.0.1: wanted: {"node":">=14.14.0"} (current: {"node":"12.22.1","npm":"6.14.12"})
npm ERR! notsup Not compatible with your version of node/npm: klaw@4.0.1
npm ERR! notsup Not compatible with your version of node/npm: klaw@4.0.1
npm ERR! notsup Required: {"node":">=14.14.0"}
npm ERR! notsup Actual:   {"npm":"6.14.12","node":"12.22.1"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-01-25T14_18_49_333Z-debug.log
root@ea327157d1ce:/src/app/temp# rm .npmrc
root@ea327157d1ce:/src/app/temp# npm install jsdoc
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN notsup Unsupported engine for klaw@4.0.1: wanted: {"node":">=14.14.0"} (current: {"node":"12.22.1","npm":"6.14.12"})
npm WARN notsup Not compatible with your version of node/npm: klaw@4.0.1
npm WARN temp@1.0.0 No description
npm WARN temp@1.0.0 No repository field.

+ jsdoc@3.6.10
added 25 packages from 26 contributors and audited 25 packages in 9.446s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

### Your environment

| Software         | Version |
| ---------------- | ------- |
| JSDoc            | `^3.6.0`
| Node.js          | `12.22.1`
| npm              | `6.14.12`
| Operating system | docker ubutntu 18.04

**Comment by BrianFreedman** [comment]
sorry... a different issue made it look like this was failing when it isn't

**Comment by philliplakis** [comment]
+1
My tests are failing due to being on `12.22.9` .

```error klaw@4.0.1: The engine "node" is incompatible with this module. Expected version ">=14.14.0". Got "12.22.9"```

**Comment by damienmutt** [comment]
+1 cannot install JSDoc because of `kraw` dependency (working on `node 12.22.0`)
`error klaw@4.0.1: The engine "node" is incompatible with this module. Expected version ">=14.14.0". Got "12.22.0"`

**Comment by duartealexf** [comment]
I ran into the same issue. The "fix" I applied was to set the `jsdoc` dependency to version 3.6.7.

From 3.6.8 onwards it uses klaw@^4, which breaks my CI.

**Comment by hegemonic** [comment]
Fixed by #2002.
